compiler-core/src/language_server/tests/snapshots/gleam_core__language_server__tests__action__pattern_match_on_argument_works_on_fn_arguments.snap

```diff
@@ -0,0 +1,25 @@
+---
+source: compiler-core/src/language_server/tests/action.rs
+expression: "\npub fn main() {\n  [#(1, 2)]\n  |> map(fn(tuple) {})\n}\n\nfn map(list: List(a), fun: fn(a) -> b) { todo }\n"
+---
+----- BEFORE ACTION
+
+pub fn main() {
+  [#(1, 2)]
+  |> map(fn(tuple) {})
+            ↑         
+}
+
+fn map(list: List(a), fun: fn(a) -> b) { todo }
+
+
+----- AFTER ACTION
+
+pub fn main() {
+  [#(1, 2)]
+  |> map(fn(tuple) {
+  let #(value_0, value_1) = tuple
+})
```

Comment:
In this case of anonymous function with an empty body inside a multiline pipe I couldn't figure out a way to print it nicely with the correct indentation that is not too messy. Do you feel like this is ok, given the formatter will take care of it anyway?

Reply:
Yeah it's fine